src/kw_remote.sh

```diff
@@ -256,6 +273,10 @@ function parse_remote_options()
         options_values['RENAME']=1
         shift
         ;;
+      list)
```

Comment:
You need to add `list` to the `long_options` variable. You also need to update the `bash_autocomplete.sh`.